ocaml-lsp-server/src/document.ml

```diff
@@ -128,22 +128,15 @@ module Single_pipeline : sig
     -> f:(Mpipeline.t -> 'a)
     -> ('a, Exn_with_backtrace.t) result Fiber.t
 end = struct
-  type t =
-    { thread : Lev_fiber.Thread.t
-    ; mutable last : (Text_document.t * Mconfig.t * Mpipeline.t) option
```

Comment:
does this mean that we don't try reusing the last pipeline if it's the same document?

Reply:
Yes, the optimization is gone. I don't think it ever worked anyway given that it relied on physical equality of the `Mconfig.t` being return.